pages/insight_menu.php

```diff
@@ -0,0 +1,67 @@
+
+<style>
+
+ul {
+    list-style: none;
+}
+
+* {
+    font-family: Arial;
+}
+
+#categoryPrice {
+    float:right;
+    font-weight: normal;
+    font-size:17px;
+}
+
+#price {
+    float:right;
+}
+
+</style>
+
+<?php
+
+function echoCategory($categoryId, $size = 1) {
+    $subcategories = base_query("SELECT * FROM DishCategory WHERE ParentCategoryId = :categoryId ORDER BY Position", [':categoryId' => $categoryId])->fetchAll();
+    $category = base_query("SELECT * FROM DishCategory WHERE Id = :categoryId", [':categoryId' => $categoryId])->fetch();
+?>
+<!-- Echo category name -->
+    <h<?= $size ?>>
```

Comment:
Willen we future proof zijn en zorgen dat hij het nog steeds goed doet als je meer dan 6 categorieën onder elkaar hebt?